pkg/occlient/occlient.go

```diff
@@ -1647,7 +1647,7 @@ func tar(tw *taro.Writer, fileName string, destFile string) error {
 	}
 	splitFileName := strings.Split(fileName, destFile)[1]
 
-	hdr.Name = destFile + splitFileName
+	hdr.Name = destFile + strings.Replace(splitFileName, "\\", "/", -1)
```

Comment:
Mind adding a small comment that this is intended for Windows / Mac OS / Linux support?